bin/carthage-bootstrap-if-needed

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+if ! diff Cartfile.resolved Carthage/Cartfile.resolved &>/dev/null; then
+  carthage bootstrap 
+  cp Cartfile.resolved Carthage
+fi
```

Comment:
I guess this isn't just for CI. I _think_ the intention must be to commit `Carthage/Cartfile.resolved` at some point?